src/tasks/MonoTargetsTasks/MonoTargetsTasks.csproj

```diff
@@ -5,36 +5,56 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <Nullable>enable</Nullable>
-    <NoWarn>$(NoWarn),CA1050</NoWarn>
+    <NoWarn>$(NoWarn),CA1050,CS0618,CS0649,CS8632,SYSLIB0003</NoWarn>
```

Comment:
Why was `SYSLIB0003` needed?

Reply:
And can we fix the ones causing CS0649, and CS0618?

Reply:
Turns out it was mostly Cecil.  Updated the warnings in both new projects.